doc/api/esm.md

```diff
@@ -1130,18 +1130,42 @@ node --experimental-top-level-await b.mjs # works
 <!-- type=misc -->
 
 To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
-
+provided via `--experimental-loader ./loader-name.mjs` arguments to Node.js.
 When hooks are used they only apply to ES module loading and not to any
 CommonJS modules loaded.
 
+Loaders are modules which export functions and data which can be used by
+Node.js to modify the behavior of loading ES Modules. Loaders may be ”chained”
+together by passing multiple `--experimental-loader` arguments to Node.js. A
+loader module will be imported once as a singleton.
+
+A hook has access to an additional `nextLoader` argument, which contains
```

Comment:
As I've said maybe four times now, default hooks will be exposed in future changes I'm planning to make.

Reply:
an example of why exposing next is useful:

```js
// security loader

export async function resolve(specifier, parentURL, context, next) {
  const url = await next.resolve(specifier, parentURL, context);
  if (isInSecurityContext(url)) {
    throw new Error('You do not have access to this resource');
  }
  return url;
}
```

Reply:
@devsnek perhaps you can share some of your plans here further for future exposing of the hooks? Typically a PR like this would have a shared collaborative design document since the hooks are such a deep piece of law.

Reply:
@guybedford I don't think I've introduced anything new... The only issue is that "next loader" and "default loader" are the same thing in the current api, and chaining separates them. Both will still be available, but only one of those makes sense as a contextual argument to a hook.